ocean_lib/models/arguments.py

```diff
@@ -18,6 +20,28 @@
 logger = logging.getLogger("ocean")
 
 
+class FeeTokenArguments:
```

Comment:
Like recommended before: this class is not only used as arguments. It's also used via getters. 

Therefore rename appropriately.

Also, move it out of the arguments module (or rename the module to something like structs.py)